### PR TITLE
PBC-394 Integrated SearchHttp module into B2B Marketplace Demoshop

### PR DESCRIPTION
**Pull request by bohdan-turchyk-spryker**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/PBC-394

###### Change log

- Integrated SearchHttp latest changes into B2B Marketplace Demoshop

### CODE REVIEW

File: src/Pyz/Client/Catalog/CatalogDependencyProvider.php [hunk 1]
```diff
@@ -208,4 +224,68 @@ protected function getProductConcreteCatalogSearchQueryExpanderPlugins(): array
...
+    /**
+     * @return array<string, array<\Spryker\Client\SearchExtension\Dependency\Plugin\QueryExpanderPluginInterface>>
+     */
+    protected function createCatalogSearchQueryExpanderPluginVariants(): array
+    {
+        return [
+            SearchHttpConfig::TYPE_SEARCH_HTTP => [
+                new BasicSearchHttpQueryExpanderPlugin(),
+                new ProductPriceSearchHttpQueryExpanderPlugin(),
+                new FacetSearchHttpQueryExpanderPlugin(),
```
Comment (matweew) [inline src/Pyz/Client/Catalog/CatalogDependencyProvider.php:268]: +  `new MerchantReferenceSearchHttpQueryExpanderPlugin(),`

**Review by matweew — CHANGES_REQUESTED** [review]

**Review by matweew — APPROVED** [review]

File: src/Pyz/Client/Catalog/CatalogDependencyProvider.php (hunk 1 above)
Comment (matweew) [reply src/Pyz/Client/Catalog/CatalogDependencyProvider.php:269]: I'm sorry but now it's not needed anymore :)

https://github.com/spryker/suite-nonsplit/pull/7744/files
but requires SearchHttp upgrade 0.3.1 >> 0.3.2


cc @Maksym-Ohnianikov

Comment (bohdan-turchyk-spryker) [reply src/Pyz/Client/Catalog/CatalogDependencyProvider.php:268]: Updated
Do we want another QA round?

Comment (matweew) [reply src/Pyz/Client/Catalog/CatalogDependencyProvider.php:268]: as you wish

File: composer.json [hunk 2]
```diff
@@ -314,6 +317,12 @@
             "test-autoload.php"
         ]
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "git@github.com:spryker-projects/acp-configuration.git"
+        }
+    ],
```
Comment (matweew) [inline composer.json:325]: ?

**Pull request merged**